src/src-admin/App.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import Login from "./views/Login";
+import Dashboard from "./Dashboard";
+
+function App() {
+  console.log("[App] sessionStorage: "+sessionStorage.getItem("email"));
+  // console.log(sessionStorage.getItem("email")+ " === 'null': ", sessionStorage.getItem("email")==='null');
```

Comment:
Assuming we don't need all of these console.logs? Would usually remove them before pushed into develop